site/guide/manage-users.qmd

```diff
@@ -53,7 +62,7 @@ To perform these next steps, you must be a group administrator or a superuser.
 
 5. On the pop-up that appears, select the roles to be assigned and click **Assign Roles to User**.
 
-## Remove roles from a user
+### Remove roles from a user
```

Comment:
This is also missing a step between 2. and 3. You need to go to **Organization** first.

Reply:
@nrichers - I tested the steps again for this one as well - I do not see the need to go to Org first?

Reply:
Retested, your docs accurately reflect the UI.